### ISSUE TITLE
Use "hand" cursor for links as a default

### ISSUE THREAD

**Issue by dlemmermann** [JPro-one/markdown-javafx-renderer#15]
I think it would make sense to use the hand cursor as the default cursor for .markdown-link elements.

**Comment by FlorianKirmaier** [comment]
True, i guess it's relevant for desktop, because on the web, the link get's the hand-cursor from the browser

**Comment by FlorianKirmaier** [comment]
<img width="111" alt="Screenshot 2021-07-06 at 17 38 39" src="https://user-images.githubusercontent.com/6547435/124628799-1f2b2200-de81-11eb-969c-47686c35383a.png">
This is how it looks. Looks to me like a layouting bug somewhere. The HBox doesn't share the available space properly, when there are multiple lines used.

**Comment by dlemmermann** [comment]
You added your last comment to the wrong ticket.
